-- a/cmake/tribits/core/installation/set_installed_group_and_permissions.cmake.in	2021-10-29 15:03:17.000000000 -0500
+++ b/cmake/tribits/core/installation/set_installed_group_and_permissions.cmake.in	2021-11-14 21:01:35.527549019 -0600
@@ -5,7 +5,7 @@
 #
 
 # Base install dir and subdirs where last element in array gives CMAKE_INSTALL_PREFIX
-set(projectInstallBaseDir "@PROJECT_SET_GROUP_AND_PERMISSIONS_ON_INSTALL_BASE_DIR@")
+set(projectInstallBaseDir "$ENV{DESTDIR}/@PROJECT_SET_GROUP_AND_PERMISSIONS_ON_INSTALL_BASE_DIR@")
 set(projectSubdirPathsArray "@PROJECT_SUBDIR_PATHS_ARRAY@")
 # Group and permissions
 set(PROJECT_MAKE_INSTALL_GROUP "@PROJECT_MAKE_INSTALL_GROUP@")
